Sources/XCBProtocol/RPC/RPCPacket.swift

```diff
@@ -37,7 +40,7 @@ public class RPCPacketCodec: ByteToMessageDecoder, MessageToByteEncoder {
     
     public func decode(context: ChannelHandlerContext, buffer: inout ByteBuffer) throws -> DecodingState {
         if let packet = try findNextPacket(buffer: &buffer) {
-            logger.trace("[\(label)] Decoded RPCPacket: \(packet)")
+            os_log("[\(self.label)] Decoded RPCPacket: \(packet)")
```

Comment:
Why the `self.label` change?

Reply:
Xcode was complaining about: `Reference to property 'label' in closure requires explicit use of 'self' to make capture semantics explicit`